test/libsolidity/semanticTests/various/copying_bytes_multiassign.sol

```diff
@@ -0,0 +1,28 @@
+contract receiver {
```

Comment:
This test should probably go into `semanticTests/array/copying/`.

Reply:
Moved to array/copying in the latest commit